frontend/src/Login/Login.js

```diff
@@ -6,6 +6,26 @@ import Button from '@material-ui/core/Button';
 import axios from 'axios';
 import './Login.css';
 
+export const login = (username, password) => {
+  if (!username || !password) {
+    alert('username and password required!');
```

Comment:
For now this is fine since we haven't worked on error messaging, but generally it is advised to not use alert().